### ISSUE TITLE
The training setting for Kalantari et al.'s dataset?

### ISSUE THREAD

**Issue by ruiliu-ai** [liuzhen03/ADNet#10]
What's the learning rate, total epochs, lr decay milestones, batch size for training Kalantari et al.'s dataset?

**Comment by liuzhen03** [comment]
For Kalantari et al.'s dataset, we set the initial learning rate to 1e-4 and the decay rate of  0.1.  We trained the model with a batch size of 16, and 200 epochs enable it to converge.
